.ci/Jenkinsfile

```diff
@@ -71,17 +67,21 @@ pipeline {
             stage('Sanity checks') {
               agent { label 'ubuntu-18.04 && immutable && docker' }
               environment {
-                GOROOT = "${env.WORKSPACE}/.gimme/versions/go${env.GO_VERSION}.linux.amd64"
-                PATH = "${env.WORKSPACE}/${env.BASE_DIR}/bin:${env.PATH}:${env.GOROOT}/bin"
+                PATH = "${env.WORKSPACE}/${env.BASE_DIR}/bin:${env.PATH}"
               }
               options { skipDefaultCheckout() }
               steps {
                 withGithubNotify(context: 'Sanity checks', tab: 'tests') {
                   deleteDir()
                   unstash 'source'
-                  dir(BASE_DIR){
-                    sh script: """.ci/scripts/install-dependencies.sh "${GO_VERSION}" """, label: 'Install dependencies'
-                    preCommit(commit: "${GIT_BASE_COMMIT}", junit: true)
+                  script {
+                    env.GO_VERSION = readFile("${env.WORKSPACE}/${env.BASE_DIR}/.go-version").trim()
+                  }
```

Comment:
```suggestion
                 setEnvVar("GO_VERSION", readFile("${env.WORKSPACE}/${env.BASE_DIR}/.go-version").trim())
```

Reply:
if you out the setEnvVaron the `Checkout` stage you do not have to set it anymore.

Reply:
Grooming time! I'll refactor it as you suggest!

![image](https://media.giphy.com/media/l3V0AYbDjTVOO4zF6/giphy.gif)